lib/utils/dataflow/src/index.js

```diff
@@ -280,12 +281,18 @@ const postError = (oid, res) => {
 // Build a list of output docs
 const buildOutputs = (itype, idoc, otype, odocs, okeys, otimes, now) => {
```

Comment:
Not sure why we're building different content when error is true, for example why are we not including the processed_id field in that case